plugins/chart.ts

```diff
@@ -0,0 +1,25 @@
+import { LineChart } from 'echarts/charts'
```

Comment:
Nuxt plugin will be applied to every page, which will cause other pages to slow down. Import these components in status page instead.